types/pages/index.d.ts

```diff
@@ -1,2 +1,5 @@
+/// <reference path="../common/$w.d.ts" />
 /// <reference path="../common/declaration.d.ts" />
-/// <reference path="./$w.d.ts" />
\ No newline at end of file
+
+declare class Buffer {
```

Comment:
Can you explain why pages need a Buffer?
Worth documenting these things, because they are not self-explanatory in code..
Consider either comments (though in dts they might have undesired side effects), a markdown file, or github issues